docs/product/exceptions/severity-score.md

```diff
@@ -0,0 +1,154 @@
+| < 1 second          | 1.2x       | Exception likely caused immediate crash |
+| 1-3 seconds         | 1.15x      | Very likely related to session end      |
+| 3-5 seconds         | 1.10x      | Possibly related to session end         |
+| 5-10 seconds        | 1.0x       | Neutral                                 |
+| > 10 seconds        | 0.8x       | Session continued normally              |
+
+#### 3. Failed Span Multiplier
+
+**Range**: 1.0x - 1.3x
+
+Exceptions that occur near the time of a failed performance span (operation) receive increased severity, as they may be
+causally related.
+
+The multiplier is based on temporal proximity:
+
+- **< 1 second**: 1.3x
+- **1-3 seconds**: 1.24x
+- **3-5 seconds**: 1.18x
+- **5-10 seconds**: 1.12x
+- **> 10 seconds**: 1.0x
+
+#### 4. Failed Network Request Multiplier
+
+**Range**: 0.8x - 1.4x
+
+Network failures near an exception can indicate the root cause. The severity adjustment depends on the type of domain
+and temporal proximity.
+
+**Domain Priority**:
+
+1. **First-party domains or document load domains**: 1.0x - 1.4x (scaled by proximity)
+   - These failures are most severe as they represent calls to endpoints owned by the application
+2. **Third-party domains**: 1.0x - 1.2x (scaled by proximity)
+   - Moderate severity for general third-party failures
+3. **Ad/Analytics domains**: 0.8x (fixed)
+   - Lower severity as these are typically non-critical to core functionality
+
+**Proximity Scaling** (for first-party and third-party):
+
+- **< 1 second**: Maximum multiplier (1.4x or 1.2x)
+- **1-3 seconds**: 80% of range
+- **3-5 seconds**: 60% of range
+- **5-10 seconds**: 40% of range
+- **> 10 seconds**: 1.0x (neutral)
+
+#### 5. User click multiplier
+
+**Range**: 1.0x - 1.15x
+
+Exceptions that occur shortly after a user click receive increased severity, as they represent issues triggered directly
+by user actions.
+
+The multiplier is based on how recently the click occurred before the exception:
+
+- **< 1 second after click**: 1.15x
+- **1-3 seconds after click**: 1.12x
+- **3-5 seconds after click**: 1.09x
+- **5-10 seconds after click**: 1.06x
+- **> 10 seconds or no click**: 1.0x
+
+## Example Calculation
+
+Here's an example of how the severity score is calculated:
+
+```text
+Base Severity: 0.60 (unhandled exception)
+
+Multipliers:
+- Node Modules: 1.0x (in application code)
+- Time to Session End: 1.15x (session ended 2 seconds later)
+- Failed Span: 1.0x (no failed spans nearby)
+- Failed Network: 1.12x (third-party domain failed 4 seconds before)
+- User Click: 1.09x (click occurred 4 seconds before)
+
+Calculation:
+0.60 × 1.0 × 1.15 × 1.0 × 1.12 × 1.09 = 0.84
+
+Final Score: 0.84
+```
+
+## Viewing Severity Scores in the Dashboard
+
+In the Embrace dashboard, exceptions are grouped by their stack trace line and displayed as a list. Each row represents
+a group of exception events that share the same location in your code.
```

Comment:
put this section before you get into the specifics of how it's calculated. any screenshots you can add?

Reply:
good point! Changed and added an screenshot!